docs/concepts/architecture/master-node-communication.md

```diff
@@ -106,3 +108,83 @@ cluster (connecting to the ssh server listening on port 22) and passes all
 traffic destined for a kubelet, node, pod, or service through the tunnel.
 This tunnel ensures that the traffic is not exposed outside of the private
 GCE network in which the cluster is running.
+
+
+
+## Configuring communication ports
+
+> A note on the difference between specifying 0.0.0.0 or 127.0.0.1 as a bind `--address`. *0.0.0.0* will bind to ALL interfaces on the host. *127.0.0.1* will ONLY bind to localhost.
```

Comment:
should this be bind `--address` or `--bind-address` per the table blow? It wasn't entirely clear to me, not sure if this is an error or just reads a bit strangely